src/httpserver.cpp

```diff
@@ -327,6 +331,11 @@ static bool HTTPBindAddresses(struct evhttp* http)
                 LogPrintf("WARNING: the RPC server is not safe to expose to untrusted networks such as the public internet\n");
             }
             boundSockets.push_back(bind_handle);
+            if (firstBindAddress.empty()) {
+                std::ostringstream addr;
```

Comment:
Avoid shadowing `addr` in outer scope :-)

Reply:
Oops, fixed.  Good catch!